flex/validation/common.py

```diff
@@ -520,3 +520,15 @@ def validate_path_to_api_path(path, paths, basePath='', context=None, **kwargs):
         raise ValidationError(str(err))
 
     return api_path
+
+
+def validate_content_type(content_type, content_types, **kwargs):
+    # TODO: is it correct to skip validation for a null content_type?
+    if content_type:
+        content_type = content_type.split(';')[0]  # only check MIME type, ignore parameters
```

Comment:
*nitpick*

Merely a style choice, but I prefer the following convention for this type of string splitting.

```python
mime_type, _, _ = content_type.partition(';')
```

Would you mind making this change along with the `octet-stream` default.